winpr/include/winpr/winuser.h

```diff
@@ -0,0 +1,48 @@
+/**
+ * WinPR: Windows Portable Runtime
+ * Messagebox definitions
+ *
+ * Copyright 2014 Thincast Technologies GmbH
+ * Copyright 2014 DI (FH) Martin Haimberger <martin.haimberger@thincast.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+
+#ifndef WINPR_ASN1_H
```

Comment:
this can go in winpr/user.h instead of adding winpr/winuser.h, and even then, you have used winpr/asn1.h as the basis for this header and forgot to change the WINPR_ASN1_H header guard.